core/modules/catalog/index.ts

```diff
@@ -1,20 +1,33 @@
+import { StorefrontModule } from '@vue-storefront/module'
 import { productModule } from './store/product'
 import { attributeModule } from './store/attribute'
 import { stockModule } from './store/stock'
 import { taxModule } from './store/tax'
 import { categoryModule } from './store/category'
-import { createModule } from '@vue-storefront/core/lib/module'
-import { beforeRegistration } from './hooks/beforeRegistration'
+import * as localForage from 'localforage'
+import UniversalStorage from '@vue-storefront/core/store/lib/storage'
+import { currentStoreView } from '@vue-storefront/core/lib/multistore'
+import Vue from 'vue'
+import { initCacheStorage } from '@vue-storefront/core/helpers/initCacheStorage'
+import { StorageManager } from '@vue-storefront/core/store/lib/storage-manager'
 
-export const KEY = 'catalog'
-export const Catalog = createModule({
-  key: KEY,
-  store: { modules: [
-    { key: 'product', module: productModule },
-    { key: 'attribute', module: attributeModule },
-    { key: 'stock', module: stockModule },
-    { key: 'tax', module: taxModule },
-    { key: 'category', module: categoryModule }
-  ] },
-  beforeRegistration
-})
+export const CatalogModule: StorefrontModule = function (app, store, router, moduleConfig, appConfig) {
+  const storeView = currentStoreView()
+  const dbNamePrefix = storeView.storeCode ? storeView.storeCode + '-' : ''
+
+  initCacheStorage('categories')
```

Comment:
We should refactor this to ‚StorageManager.set’ - to have the same syntax like above (this function does exactly the same just for different storeName). The attributes store is the only one still needed (we can remove the other ones)

Reply:
We need to remove `categories` and `products` keep only `attribbutes`